### DISCUSSION TITLE
Can't set FK as Not Null

### DISCUSSION THREAD

**Discussion by ahtned** [nocodb/nocodb#4251] [Q&A]
Recently updated to 0.98.3 and now FK / lookup columns are required. I can uncheck the NN box and save but it just resets. Any help would be appreciated, thanks!
![Capture54325234](https://user-images.githubusercontent.com/107133240/199238283-e53c9796-9484-4291-af68-31dc1e7e5a53.PNG)

**Comment by o1lab** [comment]
Do you mean to say, it was fine in version <0.98.3 ?

**Reply by ahtned** [reply]
I know it was working on 0.97.0. I updated from that to 0.98.3 and it stopped working.

**Comment by ahtned** [comment]
Updated to 0.98.4. Still not working.

**Comment by ahtned** [comment]
Found a workaround by editing the "req" column for the FK in the nc_columns_v2 table from true to false and then forcing a rebuild and deployment on digital ocean.
